server/play/model/Game.test.ts

```diff
@@ -58,18 +58,19 @@ function autoplayTrick(game: Game, time: () => number) {
   const order = (testingGetState(game) as PlayingBoardState).current_trick.players;
   for (const player of order) {
     const state = testingGetState(game) as PlayingBoardState;
-    const anyPlayerPlayedCard = (state.current_trick.trick_num === 0 && state.current_trick.cards.length === 0);
-    const cards = getCardsAllowedToPlay(state.hands[getPlayerNum(state.players, player)], state.current_trick.cards, anyPlayerPlayedCard, state.called);
+    const isFirstTrick = (state.current_trick.trick_num === 0 && state.current_trick.cards.length === 0);
```

Comment:
isFirstPlay would be a better name. AnyPlayerPlayedCard should be true in the first trick if you're on the second or greater card, but isFirstTrick, I would assume no.